cartridges/int_cloudinary/cartridge/js/cloudinaryWidgets.js

```diff
@@ -63,10 +75,10 @@ window.renderCloudinaryVideoPlayer = function () {
 
 window.makeCloudinaryImagesResponsive = function () {
     var $cldResponsiveImgTags = $('.cld-responsive');
-    var $cldEl = $('.cloudinary-data-container');
+    var $cldEl = $('.cloudinary-data-container').length > 0 ? $('.cloudinary-data-container') : $('.cloudinary-data-cloud-container');
```

Comment:
The Cloudinary gallery was not working on Quick View. It conflicted the class with PLP since the quick view is a popup modal that opens on the PLP page.

Reply:
If this is for the modal then it makes sense to use `-modal` rather than `-cloud`.

In addition, I didn't fully understand the use case. If we have 2 containers then we should iterate both of them, don't we?


finally, how will this be reflected in https://github.com/asad-rafter/cloudinary_sfcc_site_cartridge/blob/b469c8b113070a87c11ff96af3af644a9df13c8d/cartridges/int_cloudinary/cartridge/static/default/js/cloudinaryWidgets.js#L84

Reply:
When both containers exist this code ignores `.cloudinary-data-cloud-container`, which doesn't seems right.
Please check

Reply:
I have already removed this class `_.cloudinary-data-cloud-container_` now i am using iterator as you mentioned.